shoes-core/lib/shoes/oval.rb

```diff
@@ -10,9 +10,9 @@ class Oval
     STYLES = { fill: Shoes::COLORS[:black] }
 
     def create_dimensions(left, top, width, height)
-      left   ||= @style[:left]
-      top    ||= @style[:top]
-      width  ||= @style[:diameter] || @style[:width] || (@style[:radius] || 0) * 2
+      left ||= @style[:left]
+      top ||= @style[:top]
+      width ||= @style[:diameter] || @style[:width] || (@style[:radius] || 0) * 2
```

Comment:
Thanks for pointing this out, I missed this and I agree with you. Looks like there's a configuration option called `MultiSpaceAllowedForOperators` that will allow lining these up. I'll drop this commit from the PR and make a new one for this cop.